packages/spindle-ui/src/Rating/StarFront.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+interface Props extends React.SVGProps<SVGSVGElement> {}
+
+export const StarFront = (props: Props) => {
+  return (
+    <svg
+      fill="none"
```

Comment:
imo.
```suggestion
      fill="currentColor"
```
にして、CSSを https://github.com/openameba/spindle/pull/1346/files#diff-dc585571cf65c9bfce92c9a43d1ca3a284e5eaaa4a4ac20aad86b5fae4809899R55-R57
```
.spui-Rating-item--full .spui-Rating-icon,
.spui-Rating-item--active .spui-Rating-icon {
  color: var(--color-object-rating);
}
```
のように指定すれば、
https://github.com/openameba/spindle/pull/1346/files#diff-ef7d8b1dc4d8f7d6868dea70f9408a6cf023f03894ee9a0fd454810acfdd5fcaR17 のfillでカラーコード指定が不要になりそうだなと思いました 🙏

Reply:
かつ、 `.spui-Rating-item--full ` と `.spui-Rating-item--active`のスタイル指定が他の箇所含めて完全一致するので、セレクタの種類を一つ減らせるかも？（`.spui-Rating-item--active` に統一できるかも？）